File: node.js/백준/Bronze/10871. X보다 작은 수/X보다 작은 수.js
const fs = require("fs")

const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n")

const [N, X] = input[0].split(" ").map(Number)

const numbers = input[1].split(" ").map(Number)

const smallerNumbers = numbers.filter((e) => e < X)

const ans = smallerNumbers.join(" ")

console.log(ans)

File: Rust/백준/Bronze/10871. X보다 작은 수/X보다 작은 수.rs
use std::io::{self, Read};

pub fn main() {
  let mut input = String::new();
  io::stdin().read_to_string(&mut input).unwrap();
  
  let numbers: Vec<i32> = input.split_whitespace().filter_map(|s| s.parse().ok()).collect();
  
  if numbers.len() < 2 {
    return;
  }
  
  let x = numbers[1];

  let res : Vec<String>= numbers[2..].iter().filter(|&n| n < &x).map(|n| n.to_string()).collect();
  
  println!("{}", res.join(" "))
  
}